spopt/region/azp.py

```diff
@@ -436,7 +438,7 @@ def fit_from_dict(
             corresponding area is assigned to at the beginning of the
             algorithm.
             If None, then a random initial clustering will be generated.
-        objective_func : :class:`region.ObjectiveFunction`, default: ObjectiveFunctionPairwise()
```

Comment:
Why have you removed this link?

Reply:
Can't remember. I will add back and update the others in `azp`